syncano/models/archetypes.py

```diff
@@ -104,6 +110,29 @@ def __eq__(self, other):
             return self.pk == other.pk
         return NotImplemented
 
+    @classmethod
+    @contextmanager
+    def bulk(cls, instance):
```

Comment:
Instance param shouldn't be required and it should default to instance set in connection